src/messages.js

```diff
@@ -0,0 +1,11 @@
+export const search_location = "Search a Location";
```

Comment:
instead of directly creating a message js file, you can create a ts file under constants and can be named as CommonConstants.ts.
Also no need to export it like a constant parameters. instead you can create it as constant file like 
export default {
SEARCH_LOCATION : 'Search a Location'
.........
.........
}
The parameter names should be in capital letters.
ypu can refer AuthConstants.ts file as an example. The messages corresponds to Auth is added in that.